### PR TITLE
fix: color contrast a11y

### PR DESCRIPTION
**Pull request by CDharmateja**
# Issue #57 
Color changes to maintain minimum contrast of 4.5

### Changes
- Foreground color of clause header, error header, links(Import from VS Code, Upload CTA file)

### CODE REVIEW

File: src/ErrorLogger/styles.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const ErrorsHeader = styled.div`
     box-shadow: ${props => props.headerShadow || '0 -2px 20px 0 rgba(20,31,60,0.65)'};
     border-top: ${props => props.headerTop || ' 1px solid #50637F'};
 
-    color: ${props => (props.errors ? '#FF4242' : '#19C6C7')};
+    color: ${props => (props.errors ? '#FFA8A8' : '#19C6C7')};
```
Comment (irmerk) [inline src/ErrorLogger/styles.js]: This change would make us WCAG AAA compliant for normal text, but this is bolded text so `#FF6161` would be sufficient.

File: src/TemplateLibrary/Buttons/index.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ const UploadButton = styled.button`
   text-align: right;
   text-decoration: none;
   font-size: 0.87em;
-  color: #76777D;
+  color: #86878D;
```
Comment (irmerk) [inline src/TemplateLibrary/Buttons/index.js]: What is the background color at this point? I'm imagining this change is fine.

File: src/components/styles.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ export const ClauseHeader = styled.div`
   justify-self: start;
   margin: 6px 0;
   padding: 3px;
-  color: ${props => props.headercolor || '#939EBA'};
+  color: ${props => props.headercolor || '#63779C'};
```
Comment (irmerk) [inline src/components/styles.js]: The background here is gradient from `#FFFFFF` to `#ECF0FA` , so I would think something darker like `#5B6E8F` would be necessary.

File: src/components/styles.js [hunk 4]
```diff
@@ -54,7 +54,7 @@ export const DeleteWrapper = styled.div`
 `;
 
 export const ClauseDelete = styled.svg`
-  fill: #939EBA;
+  fill: #63779C;
```
Comment (irmerk) [inline src/components/styles.js]: The background here is gradient from `#FFFFFF` to `#ECF0FA` , so I would think something darker like `#5B6E8F` would be necessary.

Comment (Michael-Grover) [reply src/components/styles.js]: Let's use #505D7E

File: src/components/styles.js (hunk 3 above)
Comment (Michael-Grover) [reply src/components/styles.js]: Let's use #505D7E

File: src/ErrorLogger/styles.js (hunk 1 above)
Comment (Michael-Grover) [reply src/ErrorLogger/styles.js]: 👍

File: src/TemplateLibrary/Buttons/index.js (hunk 2 above)
Comment (Michael-Grover) [reply src/TemplateLibrary/Buttons/index.js]: 👍

Comment (CDharmateja) [reply src/TemplateLibrary/Buttons/index.js]: Background color is #141F3C. You can view new color contrast using this link https://webaim.org/resources/contrastchecker/?fcolor=86878D&bcolor=141F3C

File: src/ErrorLogger/styles.js (hunk 1 above)
Comment (CDharmateja) [reply src/ErrorLogger/styles.js]: #FF6161 is giving 2.9:1 contrast ratio (https://webaim.org/resources/contrastchecker/?fcolor=FF6161&bcolor=364C77)

File: src/components/styles.js (hunk 3 above)
Comment (CDharmateja) [reply src/components/styles.js]: #5B6E8F is indeed giving more contrast. I checked it w.r.t #FFFFFF (given as background color by wave extension). I will change this.

File: src/components/styles.js (hunk 4 above)
Comment (CDharmateja) [reply src/components/styles.js]: #5B6E8F is indeed giving more contrast. I checked it w.r.t #FFFFFF (given as background color by wave extension). I will change this.

File: src/ErrorLogger/styles.js (hunk 1 above)
Comment (irmerk) [reply src/ErrorLogger/styles.js]: Ah good point, didn't check the hover background. Leave this as you changed then.

File: src/components/styles.js (hunk 4 above)
Comment (CDharmateja) [reply src/components/styles.js]: #5B6E8F is giving me 5.15 contrast where as #505D7E is giving contrast 6.54

File: src/components/styles.js (hunk 3 above)
Comment (CDharmateja) [reply src/components/styles.js]: #5B6E8F is giving me 5.15 contrast where as #505D7E is giving contrast 6.54

File: src/components/styles.js (hunk 4 above)
Comment (irmerk) [reply src/components/styles.js]: That sounds good.

File: src/components/styles.js (hunk 3 above)
Comment (irmerk) [reply src/components/styles.js]: That sounds good.

File: src/components/styles.js (hunk 4 above)
Comment (Michael-Grover) [reply src/components/styles.js]: please make it #505D7E

File: src/components/styles.js (hunk 3 above)
Comment (Michael-Grover) [reply src/components/styles.js]: let's use #505D7E please

File: src/components/styles.js (hunk 4 above)
Comment (CDharmateja) [reply src/components/styles.js]: Okay

**Comment by irmerk** [comment]
Please add a DCO signoff and rebase to be up to date with master, @ me to notify when it's ready to review.

**Comment by irmerk** [comment]
@CDharmateja any progress on this?

**Comment by irmerk** [comment]
Closing as stale, feel free to come back to it @CDharmateja!

**Pull request closed without merging**